packages/node/src/generators/application/files/koa/src/main.ts__tmpl__

```diff
@@ -0,0 +1,11 @@
+import koa from 'koa';
+
+const app = new koa();
+
+app.use(async ctx =>{
+    ctx.body = 'Hello from Nrwl 🐳 API';
```

Comment:
Minor: Let's print something like `Hello API` -- avoid Nrwl since we're trying to promote as "Nx".

I think we also want it in JSON to mimic an API. Like `{ "message": "Hello API" }` or something.

Reply:
👍🏿